examples/omp-target-kernel.cpp

```diff
@@ -10,35 +10,40 @@
 using namespace RAJA;
 using namespace RAJA::statement;
 
-int main(int /*argc*/, char** /*argv[]*/) {
+int main(int /*argc*/, char** /*argv[]*/)
+{
 
+  // clang-format off
   // using Pol = KernelPolicy<
   //               For<1, RAJA::seq_exec>,
   //               For<0, RAJA::omp_target_parallel_for_exec<1>, Lambda<0> >
   //             >;
   using Pol = KernelPolicy<
-    Collapse<omp_target_parallel_collapse_exec, ArgList<0,1>, Lambda<0> > >;
+      // clang-format on
+      Collapse<omp_target_parallel_collapse_exec, ArgList<0, 1>, Lambda<0>>>;
```

Comment:
Here's an example of what I don't like. Putting all the closing '>' on the same line makes it harder to parse visually. I prefer that the indentation pattern that we follow consistently for the most part, where opening '<' and 'closing '>' are vertically aligned in execution policies.